### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by amy-silcock**
Fix broken links in Developer Documentation. Most caused by the migration to wagtail, others have been present for a long time.

### CODE REVIEW

File: codelist-api.rst [hunk 1]
```diff
@@ -164,5 +164,5 @@ If you do this you may not need to alter the way you deal with codelists, but th
 * Version information is no longer provided in the XML.
 * Specific changes to codelists:
 
-  * :doc:`/codelists/BudgetIdentifier/` has different categories, and no 'sector' elements.
-  * Names in the :doc:`/codelists/FileFormat/` list are no longer there as they make little sense, and the list now tracks the IANA source it is derived from directly.
+  * The `BudgetIdentifier <https://github.com/orgs/IATI/teams>`__codelist has different categories, and no 'sector' elements.
```
Comment (andylolz) [inline codelist-api.rst:167]: I don’t think this link is correct https://github.com/orgs/IATI/teams

File: codelist-api.rst [hunk 2]
```diff
@@ -164,5 +164,5 @@ If you do this you may not need to alter the way you deal with codelists, but th
...
+  * Names in the the `FileFormat <https://iatistandard.org/en/iati-standard/203/codelists/fileformat/>`__ codelist are no longer there as they make little sense, and the list now tracks the IANA source it is derived from directly.
```
Comment (andylolz) [inline codelist-api.rst:168]: > the the

^^ typo here

File: old-datastore/reference/data-api.rst [hunk 3]
```diff
@@ -45,7 +45,7 @@ Example API call:
 reporting-org
 `````````````
 
-Returns activities where the `reporting-org <http://iatistandard.org/activity-standard/iati-activities/iati-activity/reporting-org/>`__ element has a @ref attribute value that matches your specified `Organisation identifier <http://iatistandard.org/organisation-identifiers/>`__ string.
+Returns activities where the `reporting-org <http://iatistandard.org/activity-standard/iati-activities/iati-activity/reporting-org/>`__ element has a @ref attribute value that matches your specified `Organisation identifier <https://iatistandard.org/en/iati-standard/203/activity-standard/iati-activities/iati-activity/reporting-org/>`__ string.
```
Comment (andylolz) [inline old-datastore/reference/data-api.rst:48]: The link on “Organisation identifier” here points to [reporting-org](https://iatistandard.org/en/iati-standard/203/activity-standard/iati-activities/iati-activity/reporting-org/). Is that intended?

File: codelist-api.rst (hunk 1 above)
Comment (andylolz) [reply codelist-api.rst:167]: Also, I think there’s a missing space here:

> __codelist

Comment (amy-silcock) [reply codelist-api.rst]: Thanks Andy - I've corrected these

File: old-datastore/reference/data-api.rst (hunk 3 above)
Comment (amy-silcock) [reply old-datastore/reference/data-api.rst:48]: Yes

Comment (andylolz) [reply old-datastore/reference/data-api.rst:48]: So the sentence will read like this:

> #### reporting-org
> Returns activities where the [reporting-org](http://iatistandard.org/activity-standard/iati-activities/iati-activity/reporting-org/) element has a `@ref` attribute value that matches your specified [Organisation identifier](https://iatistandard.org/en/iati-standard/203/activity-standard/iati-activities/iati-activity/reporting-org/) string.

…where both links point to the same page.

Intuitively, I’d expect these links to go to different places.

Comment (amy-silcock) [reply old-datastore/reference/data-api.rst:48]: Agreed. If you would like, please do re-write/update the documentation and send in a PR.

At the moment I'm just fixing links and aligning them to the new content/structure. From this point on we're not going to be improving the guidance for the old datastore. But happy for you to do so.

Comment (andylolz) [reply old-datastore/reference/data-api.rst:48]: I can’t tell if we’re speaking at cross-purposes.

The link previously pointed here:
http://iatistandard.org/organisation-identifiers/

I think it should have been changed to point here:
http://iatistandard.org/203/organisation-identifiers/

Instead, this PR changed it to point here:
https://iatistandard.org/en/iati-standard/203/activity-standard/iati-activities/iati-activity/reporting-org/

That’s not the same intended location, so I don’t think it fixes the link.

**Pull request merged**